README.md

```diff
@@ -105,6 +105,18 @@ The following image types are currently available via the `--type` argument:
 | `ami`                 | [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) |
 | `qcow2` **(default)** | [QEMU](https://www.qemu.org/)                                                         |
 
+### ☁️ AMI flags
```

Comment:
I think we also need to document that you need to pass the secrets via environment flags. I wonder if it's worth having a new section for uploaders...

Reply:
Ah you're right.  Credentials were on my mind while I was writing it and it somehow slipped out by the end.

Reply:
Done.  Take a look :)